public/app/features/admin/UpgradePage.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import { css } from 'emotion';
+import { NavModel } from '@grafana/data';
+import Page from '../../core/components/Page/Page';
+import { LicenseChrome } from './LicenseChrome';
+import { Forms } from '@grafana/ui';
+import { hot } from 'react-hot-loader';
+import { StoreState } from '../../types';
+import { getNavModel } from '../../core/selectors/navModel';
+import { connect } from 'react-redux';
+
+interface Props {
+  navModel: NavModel;
+}
+
+export const UpgradePage: React.FC<Props> = ({ navModel }) => {
+  return (
+    <Page navModel={navModel}>
+      <Page.Contents>
+        <UpgradeInfo
+          editionNotice="You are running the open-source version of Grafana.
+        You have to install the Enterprise edition in order enable Enterprise features."
+        />
+      </Page.Contents>
+    </Page>
+  );
+};
+
+const titleStyles = { fontWeight: 500, fontSize: '26px', lineHeight: '123%' };
+
+interface UpgradeInfoProps {
+  editionNotice?: string;
+}
+
+export const UpgradeInfo: React.FC<UpgradeInfoProps> = ({ editionNotice }) => {
+  return (
+    <LicenseChrome header="Grafana Enterprise" subheader="Get your free trial" editionNotice={editionNotice}>
+      <FeatureInfo />
+      <ServiceInfo />
+    </LicenseChrome>
+  );
+};
+
+const GetEnterprise: React.FC = () => {
+  return (
+    <div style={{ marginTop: '60px', marginBottom: '120px' }}>
+      <h2 style={titleStyles}>Get Grafana Enterprise</h2>
+      <CallToAction />
+      <p style={{ paddingTop: '12px' }}>
+        You can use the trial version for free for <strong>30 days</strong>. We will remind you about it
+        <strong>5 days before the trial period ends</strong>.
+      </p>
+    </div>
+  );
+};
+
+const CallToAction: React.FC = () => {
+  return (
+    <Forms.Button
+      variant="primary"
+      size="lg"
+      onClick={() =>
+        (window.location.href = 'https://grafana.com/contact?about=grafana-enterprise&utm_source=grafana-upgrade-page')
```

Comment:
@torkelo we have link button component. Is it better to use it here instead of invoking `window.location.href`?
```suggestion
    <LinkButton
      variant="primary"
      size="lg"
      href="https://grafana.com/contact?about=grafana-enterprise&utm_source=grafana-upgrade-page"
    >
      Contact us and get a free trial
    </LinkButton>
```

Reply:
Not sure if there's any reason why the `LinkButton` in `Forms` wasn't exposed, but it seems like the Forms buttons are different in (at least) colors from the regular buttons, so I exposed the LinkButton from Forms and used that. Might not be what we want, if so lmk

Reply:
I think it was just a mistake (not exported LinkButton).